docs/general/kusama/kusama-statemine.md

```diff
@@ -0,0 +1,86 @@
+---
+id: kusama-statemine
+title: Statemine
+sidebar_label: Statemine
+description: Statemine and its features
+slug: ../../kusama-statemine
+---
+
+## Statemine
+
+Statemine is a generic assets parachain which provides functionality for deploying and transferring assets — both Fungible and Non-Fungible Tokens (NFTs). It is a common good parachain on Kusama (not to be confused with [Statemint](../../learn/learn-common-good-chains.md#statemint), which is a parachain on Polkadot). The native token of Statemine is KSM. Balance transfers and the Existential Deposit (ED) of KSM on Statemine are about 1/10th of the values on the Relay chain. 
+Apart from the core protocol token KSM, the assets held on Statemine can be broadly categorized as 
+
+- Assets backed by an on-chain protocol’s utility
+- Assets with off-chain backing
+- Assets without any backing
+
+## Creating Assets on Statemine
+
+::: info
+
+Before minting assets on Statemine, we recommend that you try out this tutorial on Westmint, which is a parachain on Westend. 
+The WND tokens (Westies) are free and are available through a [faucet](https://wiki.polkadot.network/docs/learn-DOT#getting-westies).
+
+:::
+
+To create an asset on Statemine, you would need a deposit of 1 KSM. Ensure that your Statemine account balance more than 1 KSM,
```

Comment:
```suggestion
To create an asset on Statemine, you would need a deposit of 1 KSM. Ensure that your Statemine account balance is more than 1 KSM,
```

Reply:
This process requires a bit more than 1 KSM because it also registers the asset metadata in the batch, which requires a small storage deposit.